kubernetes-examples/src/main/java/io/fabric8/kubernetes/examples/kubectl/equivalents/CreateResourceQuotaInNamespaceYamlEquivalent.java

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (C) 2015 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.fabric8.kubernetes.examples.kubectl.equivalents;
+
+import io.fabric8.kubernetes.api.model.Quantity;
+import io.fabric8.kubernetes.api.model.ResourceQuota;
+import io.fabric8.kubernetes.api.model.ResourceQuotaBuilder;
+import io.fabric8.kubernetes.client.KubernetesClient;
+import io.fabric8.kubernetes.client.KubernetesClientBuilder;
+
+/**
+ * This sample code is Java equivalent to `kubectl create -f resources.yaml --namespace=myspace`.
+ * It loads YAML manifest with specified name and creates it.
```

Comment:
This javadoc description doesn't align with what code is doing. It says YAML manifest gets loaded but in code we're actually creating an object inline.

Either this needs to be changed or you need to update code to use `.load()` method to load ResourceQuota provided in `resources.yaml` file

Reply:
Also you'd need to add an entry in this table for this file https://github.com/fabric8io/kubernetes-client#kubectl-java-equivalents